internal/store.go

```diff
@@ -472,6 +472,29 @@ func (s *Store[K, V]) Len() int {
 	return total
 }
 
+func (s *Store[K, V]) Size() int {
```

Comment:
I'm just going to express my unnecessary opinion.

Unfortunately, this is still only an estimated size, which may change during the execution of this method.

I don't think anyone would expect an exact size from a concurrent data structure, since
1. The scheduler can delay execution.
2. The size may change in a nanosecond.
3. I can't imagine scenarios where this is needed.

Reply:
Reasonable, the size might change significantly before and after looping. @nlachfr , do you have a real use case for this? If not, this method might not be needed, and adding it could cause some confusion/issue.

Reply:
Not really, the `EstimatedSize` implementation suits my use cases
I will remove the `Size` method then